src/main/scala/viper/gobra/frontend/InformativeErrorListener.scala

```diff
@@ -77,15 +76,14 @@ class InformativeErrorListener(val messages: ListBuffer[ParserError], val source
     *
     * @see [[FailedPredicateException]]
     * @param context
-    * @param exception
     * @return
     */
-  def analyzeFailedPredicate(implicit context: ParserErrorContext, exception: FailedPredicateException): ErrorType = {
+  def analyzeFailedPredicate(implicit context: ParserErrorContext): ErrorType = {
```

Comment:
I'm not sure about this change here, because the exception is the only parameter that somewhat bounds this error handler to the mentioned error. Maybe an unused annotation would be better suited?